File: src/iniciante/Bee1008.java
package iniciante;

import java.util.Scanner;

public class Bee1008 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int funcionarios = input.nextInt();
        int valorHora = input.nextInt();
        float horaTrabalhada = input.nextFloat();

        System.out.println(String.format("NUMBER = %d\nSALARY = U$ %.2f", funcionarios, (horaTrabalhada*valorHora)));
    }
}

File: iniciante/bee1008.py
def wage (valor_hora, quant_hora):
    return valor_hora*quant_hora

quant_funcionarios = int(input())
valor_hora = int(input())
quant_hora = float(input())

print(f"NUMBER = {quant_funcionarios}\n"
      f"SALARY = U$ {wage(valor_hora, quant_hora):.2f}")
